src/include/uapi/ipc/control.h

```diff
@@ -142,6 +142,9 @@ struct sof_ipc_ctrl_data {
 	struct sof_ipc_reply rhdr;
 	uint32_t comp_id;
 
+	/* for large messages sent in parts */
+	uint32_t msg_id;
```

Comment:
When updating ABI, please do not change position of current fields or insert new fields outwith the reserved fields. This change should be inserted before reserved[] and reserved array size decremented.

Reply:
yes of course. I don't get why I missed the reserved fields in this struct, stupidity I guess. will change